test/io/serializers/arrays.little.good

```diff
@@ -199,11 +199,11 @@ SUCCESS: [domain(4,int(64),one)] int(64)
 SUCCESS: [domain(5,int(64),one)] int(64)
 --- array of arrays ---
 ===== writing: =====
-1 2 3 2 4 6 3 6 9
+1.11235 1.21235 1.31235 2.11235 2.21235 2.31235 3.11235 3.21235 3.31235
 --------------------
-010000000000000002000000000000000300000000000000020000000000000004000000000000000600000000000000030000000000000006000000000000000900000000000000
+068f8bfb2accf13f9f282595c465f33f39c2be2e5efff43f83c7c57d15e600405094924ae2b201401c615f17af7f024083c7c57d15e608405094924ae2b209401c615f17af7f0a40
```

Comment:
This and the .object.good file look off to me.  Should they be in hex?

Reply:
I believe these are in hex; but they are showing the raw bits of the floating point values, so for example, the last 16 characters of this line: `0x1c615f17af7f0a40` represent `3.3123456789`

Reply:
Wild.  I'm not sure how to evaluate if that's right or not, so I'll just defer to your judgment on it

Reply:
I just double checked these with an online hex->double converter and they appear to be correct.